### DISCUSSION TITLE
INV Numbering not working

### DISCUSSION THREAD

**Discussion by reaksiproton** [boxbilling/boxbilling#1400] [Q&A] [answered]
new/fresh installation
i start fresh installation
and chnage the invoice prefix to INV
and invoice number to 2000
but. at unpaid invoice it still count from 0...
when i click the invoice as paid then the invoice number changed to 2001

any solution about this ?

**Comment by quicklyweb** [comment]
Invoice is proforma until it gets paid. Once it is paid then it becomes an
invoice. Then a number is attached to it.

This message contains information intended solely for the addressee, which
is confidential or private in nature. If you are not the intended
recipient, you may not peruse, use, disseminate, distribute or copy this
message or any file attached to this message. Any such unauthorised use, is
prohibited and may be unlawful. If you have received this message in error,
please notify the sender immediately by e-mail or telephone and thereafter
delete the original message from your machine.

**Reply by jaapmarcus** [reply]
> Invoice is proforma until it gets paid. Once it is paid then it becomes an invoice. Then a number is attached to it. This message contains information intended solely for the addressee, which is confidential or private in nature. If you are not the intended recipient, you may not peruse, use, disseminate, distribute or copy this message or any file attached to this message. Any such unauthorised use, is prohibited and may be unlawful. If you have received this message in error, please notify the sender immediately by e-mail or telephone and thereafter delete the original message from your machine.
> […](#)

This is stupid.  "Clients" usually pay with the invoice number as reference. As soon as an invoice has been send to the client the invoice number may never change / alter. 

At least in EU this is the case.

**Reply by reaksiproton** [reply]
so this is normal ? 
because i see on clientexec whmcs and hostbill they have same inv number  between unpaid and paid.....

**Comment by quicklyweb** [comment]
Okay. So, in accounting and this is standard in UK as well as in EU. You
don't have to send customers an invoice first. You can generate invoice on
post payment basis. Like buying a stuff in supermarket. The moment you pay
to cashier she generates invoice. So proforma works the same way. It is
practically a quote you send the customer to accept and pay.

This message contains information intended solely for the addressee, which
is confidential or private in nature. If you are not the intended
recipient, you may not peruse, use, disseminate, distribute or copy this
message or any file attached to this message. Any such unauthorised use, is
prohibited and may be unlawful. If you have received this message in error,
please notify the sender immediately by e-mail or telephone and thereafter
delete the original message from your machine.

**Comment by quicklyweb** [comment, marked as answer]
Not sure about client exec but whmcs does proforma invoices too.

This message contains information intended solely for the addressee, which
is confidential or private in nature. If you are not the intended
recipient, you may not peruse, use, disseminate, distribute or copy this
message or any file attached to this message. Any such unauthorised use, is
prohibited and may be unlawful. If you have received this message in error,
please notify the sender immediately by e-mail or telephone and thereafter
delete the original message from your machine.

**Comment by quicklyweb** [comment]
Other issue you need to consider is that paying vat on unpaid invoices .
Then write off as bad debt to claim back. So it is long winded. Proforma
invoices will save you all that hassle

This message contains information intended solely for the addressee, which
is confidential or private in nature. If you are not the intended
recipient, you may not peruse, use, disseminate, distribute or copy this
message or any file attached to this message. Any such unauthorised use, is
prohibited and may be unlawful. If you have received this message in error,
please notify the sender immediately by e-mail or telephone and thereafter
delete the original message from your machine.

**Comment by reaksiproton** [comment]
so well.... likely i can understand your explaination
thankyou,
